### PR TITLE
Update find-court-tribunal-service-gov-uk.yml

### PR DESCRIPTION
**Pull request by linusnorton**
Add admin subdomain

### CODE REVIEW

File: environments/prod/find-court-tribunal-service-gov-uk.yml [hunk 1]
```diff
@@ -16,3 +16,6 @@ cname:
   - name: "_7c539238d4da39d0fa4242e862930d02.admin"
     ttl: 300
     record: "CA97F4612709C22E162E35D39F71B6A8.32CBBA599F56C6E44500A1CFFF18F733.d85688fce613669775c5.comodoca.com."
+  - name: "admin"
```
Comment (timja) [inline environments/prod/find-court-tribunal-service-gov-uk.yml:19]: needs cdnverify and afdverify records for this to work

Comment (linusnorton) [reply environments/prod/find-court-tribunal-service-gov-uk.yml:19]: okay, I'll get @damongreen123 to pick up this branch as part of implementing the shutter

Comment (damongreen123) [reply environments/prod/find-court-tribunal-service-gov-uk.yml:19]: Added need to verify name

File: environments/prod/find-court-tribunal-service-gov-uk.yml [hunk 2]
```diff
@@ -16,3 +16,12 @@ cname:
...
+    ttl: 300
+    record: "hmcts-prod.azurefd.net"
+ - name: "cdnverify.www"
+    ttl: 300
+    record: "cdnverify.hmcts-find-court-tribunal-shutter-prod.azureedge.net"
```
Comment (timja) [inline environments/prod/find-court-tribunal-service-gov-uk.yml:24]: probably would have admin in the name but not 100% if it fails you should get the error message with what it needs to be:
```suggestion
    record: "cdnverify.hmcts-admin-find-court-tribunal-shutter-prod.azureedge.net"
```

**Review by timja — APPROVED** [review]

**Review by cakeben — APPROVED** [review]

**Pull request merged**